nasa-gibs/worldview
# Smart Handoffs - Permalinks & Download size

## Description

Fixes #3233 #3376.

* Adds permalink support for smart handoffs
* Adds download size indication for granule selection

## How To Test

1. Open Worldview
2. Open layer picker
3. Add several layers with downloadable data (AOD and SST are good choices)
4. Move to the "Data" tab
5. Select a layer and confirm that the granule count and size displays
6. Check the "area of interest" box
7. Confirm that the granule count and size updates
8. Look at the URL, confirm that the `sh` param is set with the layer ID and concept ID (e.g `sh=MODIS_Aqua_Aerosol,C1443533683-LAADS`
9. Refresh the page
10. Confirm that the data tab is selected
11. Confirm that the same layer and collection are selected 
12. Confirm that the granule count and size are showing and match what you saw before page refresh
